LICENSE

```diff
@@ -1,6 +1,6 @@
 MIT License
 
-Copyright (c) 2020,2021,2022 awawa-dev
+Copyright (c) 2020,2021,2022,2023 awawa-dev
```

Comment:
Although this line could be written like this:
`Copyright (c) 2020-2023 awawa-dev`